nonblock-rpc/rust/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
+
+members = [
+    "core",
+    "uniffi-bindgen",
+]
```

Comment:
This setup creates nested Rust workspaces, [which is generally discouraged](https://www.reddit.com/r/rust/comments/1b1lgkw/nested_workspace_how_can_you_work_around_the/). 

Do you think there's a way to refactor this? Perhaps `uniffi-bindgen` could be included as a binary within the same crate.